libshaderc/include/shaderc.h

```diff
@@ -141,6 +142,56 @@ void shaderc_compile_options_set_disassembly_mode(
 void shaderc_compile_options_set_forced_version_profile(
     shaderc_compile_options_t options, int version, shaderc_profile profile);
 
+// When including a file, libshaderc initiates a query to get the full path
```

Comment:
Why "the full path"? There is no guarantee that a particular includer even has the notion of a path.